packages/node/src/lib/novu.ts

```diff
@@ -1,23 +1,24 @@
 import { EventEmitter } from 'events';
 import axios, { AxiosInstance } from 'axios';
 import { Subscribers } from './subscribers/subscribers';
-import {
-  TriggerRecipientsType,
-  ITriggerPayloadOptions,
-} from './subscribers/subscriber.interface';
+import { ITriggerPayloadOptions } from './subscribers/subscriber.interface';
 
 export class Novu extends EventEmitter {
   private readonly apiKey?: string;
   private readonly http: AxiosInstance;
   readonly subscribers: Subscribers;
 
-  constructor(apiKey: string) {
+  constructor(apiKey: string, config?: INovuConfiguration) {
     super();
 
+    const backendUrl = config?.backendUrl
+      ? config?.backendUrl
```

Comment:
What do you think, should we append the `v1` at the end of the URL?

```suggestion
      ? config?.backendUrl + '/v1'
```

This way the SDK can manage different versions for himself.